sdk-springboot/src/main/java/io/dapr/springboot/DaprRuntime.java

```diff
@@ -58,24 +59,26 @@ public static DaprRuntime getInstance() {
   /**
    * Adds a topic to the list of subscribed topics.
    *
-   * @param pubsubName Pubsub name to subcribe to.
+   * @param pubSubName PubSub name to subscribe to.
    * @param topicName Name of the topic being subscribed to.
    * @param match Match expression for this route.
    * @param priority Priority for this match relative to others.
    * @param route Destination route for requests.
    * @param metadata Metadata for extended subscription functionality.
+   * @param bulkSubscribe Bulk Subscribe configuration.
    */
-  public synchronized void addSubscribedTopic(String pubsubName,
```

Comment:
Create an overloaded method with the previous signature too to avoid a breaking change.